context/app/static/js/pages/Dataset/Dataset.jsx

```diff
@@ -96,6 +98,18 @@ function DatasetDetail(props) {
   // TODO: When all environments are clean, data_types array fallbacks shouldn't be needed.
   return (
     <DetailContext.Provider value={{ display_doi, uuid, mapped_data_access_level }}>
+      {!isLatest && (
+        <Alert severity="warning">
+          <span>
```

Comment:
Do you need the span?

Reply:
Without the span, this gets broken into multiple lines. There is almost certainly a better way to do this!